tracer/src/Datadog.Trace/Activity/DuckTypes/IActivity.cs

```diff
@@ -29,7 +29,7 @@ internal interface IActivity : IDuckType
 
         TimeSpan Duration { get; }
 
-        string? OperationName { get; }
+        string OperationName { get; }
```

Comment:
Moving from nullable to non-nullable is normally pretty balsy - even if it _looks_ like it could be non-nullable, it's rarely worth the risk of marking it as such in duck types, given it's dependent on everything _around_ that in code we don't control